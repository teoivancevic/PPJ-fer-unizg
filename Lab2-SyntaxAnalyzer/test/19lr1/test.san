%V <S> <A> <B>
%T a b
%Syn a
<S>
 <A> a <A> b
 <B> b <B> a
<B>
 $
<A>
 $
