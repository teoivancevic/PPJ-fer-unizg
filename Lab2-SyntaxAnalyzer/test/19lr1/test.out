<S>
 <A>
  $
 a 1 a
 <A>
  $
 b 1 b
